packages/contracts-bedrock/src/L2/L1BlockInterop.sol

```diff
@@ -106,4 +69,43 @@ contract L1BlockInterop is L1Block {
     function dependencySetSize() external view returns (uint8) {
         return uint8(dependencySet.length);
     }
+
+    /// @notice Sets static configuration options for the L2 system. Can only be called by the special
+    ///         depositor account.
+    /// @param _type  The type of configuration to set.
+    /// @param _value The encoded value with which to set the configuration.
+    function setConfig(ConfigType _type, bytes calldata _value) external {
```

Comment:
I recommend using the following library: https://github.com/Vectorized/solady/blob/main/src/utils/EnumerableSetLib.sol

Reply:
for what were you thinking of using that library? I can see how it could be used for managing the dependency set as a set, but not clear on how we could use it for the `ConfigType` enum

Reply:
It can work really well for maintaining the dependency set